aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -80,14 +113,19 @@ embedding_bag_cpu(const Tensor &weight, const Tensor &indices__,
   checkScalarType("embedding_bag", offsets_arg, kLong);
   Tensor indices = indices__.contiguous();
   Tensor offsets = offsets__.contiguous();
+  auto weight_arg = TensorArg(weight, "weight", 1);
+  checkScalarTypes("embedding_bag", weight_arg, {kFloat, kDouble});
```

Comment:
Is this function supposed to support half types?

Reply:
This function is CPU-only so I guess not